pkg/asset/installconfig/platformcredscheck.go

```diff
@@ -59,7 +60,7 @@ func (a *PlatformCredsCheck) Generate(dependencies asset.Parents) error {
 		opts := new(clientconfig.ClientOpts)
 		opts.Cloud = ic.Config.Platform.OpenStack.Cloud
 		_, err = clientconfig.GetCloudFromYAML(opts)
-	case libvirt.Name, none.Name, vsphere.Name:
+	case baremetal.Name, libvirt.Name, none.Name, vsphere.Name:
```

Comment:
should we be checking bmc access here?

Reply:
I think that's a good point, we probably should.  Can we address in a follow-up? I've filed https://github.com/openshift-metal3/kni-installer/issues/162 to track this.

Reply:
yeah, that would be a nice enhancement